skyVegasTests/realityCheckGameClient.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { test, type Page } from "@playwright/test";
+import { launchGame, validateConsoleMessages, login } from "./utilities";
+import * as fs from "fs";
+import { startEventListener } from "./utilities/starteventListenerHelper";
+import { makeSpin } from "./utilities/makeSpinHelper";
+
+test("realityCheck", async ({ page }) => {
+    const data = fs.readFileSync("ExpectedSlotConsoleMessages.json", "utf-8");
+    const expectedMessages = JSON.parse(data);
+    let consoleMessages: string[] = [];
+
+    await login(page);
+
+    startEventListener(page, consoleMessages);
+
+    await launchGame(page, "Big Bass: Hold & Spinner", consoleMessages);
+
+    await makeSpin(page, expectedMessages.startSpin, consoleMessages);
+
+    await page.waitForTimeout(60000)
```

Comment:
if this test is sequential, might be worth looking at getting it parallel and also being one of the first ones in the order of running (for all of these reality check waiting ones)

Reply:
^ for a future spike